packages/web/pages/user/my-account/my-orders.js

```diff
@@ -0,0 +1,348 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'react-i18next';
+import { FiMessageSquare, FiEye, FiX } from 'react-icons/fi';
+import { UserProfile } from '../../../components/UserProfile';
+import { Protected } from '../../../components/Authorization';
+import { Title } from '../../../components/Common';
+import { DataGrid } from '../../../components/DataGrid';
+import { IconButton } from '../../../components/Button';
+import { ORDERING as orderEnum } from '../../../utils/enums/api.enum';
+import { dateToString } from '../../../utils/helper';
+import { STATUS as dealStatusEnum } from '../../../utils/enums/orders.enum';
```

Comment:
```suggestion
import { ORDERING as orderEnum, STATUS as dealStatusEnum } from '../../../utils/enums;
import { dateToString } from '../../../utils/helper';
```

Reply:
We don't have an enum index file, it sounds more verbose this way, what do you think?

![image](https://user-images.githubusercontent.com/13091635/98807661-c0003500-23f9-11eb-939b-5cf0750a334d.png)